site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -2,370 +2,107 @@
 layout: "layouts/doc-post.njk"
 title: "Architecture overview"
 date: 2012-09-18
-updated: 2022-11-02
-description: A high-level explanation of the software architecture of Chrome Extensions.
-subhead: A high-level explanation of the components and structure of a Chrome Extension.
+updated: 2022-12-16
+description: A high-level explanation of the architecture of Chrome Extensions.
+subhead: A high-level explanation of the structure of a Chrome Extension.
 ---
 
-Extensions are zipped bundles of HTML, CSS, JavaScript, images, and other files used in the web
-platform. Extensions can modify web content that users see and interact with. They can also extend
-and change the behavior of the browser itself. 
+## Overview {: #overview }
 
-This page briefly describes the files that could form part of an extension, how to access these
-files, how to use Chrome APIs, how extension files communicate, and how to store
-data.
+A Chrome extension is composed of different parts. This page describes the structure of an extension, the role each part plays and how they work together. Implementation details are out of the scope of this article.
```

Comment:
The primary purpose of this rewrite was to narrow the scope of this overview to focus on structure and relation between extension parts. I moved implementation details to other pages that cover specific topics.